src/pages/store/item/[uuid].tsx

```diff
@@ -0,0 +1,93 @@
+import { Navbar } from '@/components/store';
+import AddCartButton from '@/components/store/AddCartButton';
+import ItemHeader from '@/components/store/ItemHeader';
+import SizeSelector from '@/components/store/SizeSelector';
+import { StoreAPI } from '@/lib/api';
+import config from '@/lib/config';
+import withAccessType from '@/lib/hoc/withAccessType';
+import { CookieService, PermissionService } from '@/lib/services';
+import { PrivateProfile, PublicMerchItemWithPurchaseLimits } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import { getDefaultMerchItemPhoto } from '@/lib/utils';
+import styles from '@/styles/pages/StoreItemPage.module.scss';
+import { GetServerSideProps } from 'next';
+import Image from 'next/image';
+import { useState } from 'react';
+
+interface ItemPageProps {
+  user: PrivateProfile;
+
+  item: PublicMerchItemWithPurchaseLimits;
+}
+const StoreItemPage = ({ user: { credits }, item }: ItemPageProps) => {
+  const [size, setSize] = useState<string | undefined>(item.options.length <= 1 ? 'Y' : undefined);
+  const [inCart, setInCart] = useState<boolean>(false);
+  const [amount, setAmount] = useState<number>(1);
+
+  const currOption =
+    item.options.length <= 1
+      ? item.options[0]
+      : item.options.find(val => val.metadata?.value === size);
+
+  return (
+    <div className={styles.navbarBodyDiv}>
+      <Navbar balance={credits} showBack />
+      <div className={styles.rowContainer}>
+        <div className={styles.coverContainer}>
+          <Image
+            style={{ objectFit: 'contain' }}
+            src={getDefaultMerchItemPhoto(item)}
+            alt={`Picture of ${item.itemName}`}
+            fill
+          />
+        </div>
+        <div className={styles.optionsContainer}>
+          <ItemHeader itemName={item.itemName} cost={currOption?.price} />
+          {item.options.length > 1 ? (
+            <SizeSelector currSize={size} onSizeChange={setSize} options={item.options} />
+          ) : null}
+
+          <AddCartButton
+            inCart={inCart}
+            onCartChange={setInCart}
+            currSize={size}
+            inStock={currOption?.quantity != null && currOption?.quantity >= 1}
+            lifetimeRemaining={item.lifetimeRemaining}
+            monthlyRemaining={item.monthlyRemaining}
+            amountToBuy={amount}
+            onAmountChange={setAmount}
+          />
+          <h4>Item Description</h4>
+          <p>{item.description}</p>
```

Comment:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/de8a8388-47f9-41c3-9cde-782f00a19a31)

You should add `white-space: pre-wrap;` so newline characters are kept. You might also want to increase `line-height` to, say, `1.5`, so the text doesn't look as dense:

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/37ccf179-3c64-4d07-82ee-fe22212aae68)